client/src/modules/list/components/Items/ListItemDescription/ListItemDescription.jsx

```diff
@@ -85,7 +80,6 @@ class ListItemDescription extends PureComponent {
 
   handleFocus = () => {
     const { onFocus } = this.props;
-
```

Comment:
why this change

Reply:
onFocus and onBlur were invoked twice and locks were set to true (or false) twice because socket events were emitting twice; I had problems with these behaviours so I removed those lines and realized that app is still fine but events aren't emitting twice...

Reply:
no no, I meant - why did you remove that empty line :D

Reply:
ok, I added that line